multicluster/cmd/multicluster-controller/controller.go

```diff
@@ -157,25 +158,24 @@ func setupManagerAndCertController(isLeader bool, o *Options) (manager.Manager,
 	}
 
 	if o.SelfSignedCert {
-		o.options.CertDir = selfSignedCertDir
+		o.options.Metrics.CertDir = selfSignedCertDir
 	} else {
-		o.options.CertDir = certDir
+		o.options.Metrics.CertDir = certDir
 	}
```

Comment:
I am not sure that's correct. The historic `CertDir` option seems to have been used for both the metrics server and for the webhook server. Do we need to configure `WebhookServer.CertDir` as well for the manager?

cc @luolanzone

Reply:
Yes @antoninbas is right, the historic CertDir is for webhook according to the comment. @hjiajing please help to double check and refine this part with correct fields. Thanks.